packages/bpk-component-input/readme.md

```diff
@@ -28,19 +28,21 @@ export default () => (
 
 ## Props
 
-| Property     | PropType             | Required  | Default Value    |
-| ------------ | -------------------- | --------- | ---------------- |
-| id           | string               | true      | -                |
-| name         | string               | true      | -                |
-| value        | string               | true      | -                |
-| type         | INPUT_TYPES (one of) | false     | INPUT_TYPES.TEXT |
-| valid        | bool                 | false     | null             |
-| large        | bool                 | false     | false            |
-| docked       | bool                 | false     | false            |
-| dockedFirst  | bool                 | false     | false            |
-| dockedMiddle | bool                 | false     | false            |
-| dockedLast   | bool                 | false     | false            |
-| inputRef     | func                 | false     | null             |
+| Property         | PropType             | Required            | Default Value    |
+| ---------------- | -------------------- | ------------------- | ---------------- |
+| id               | string               | true                | -                |
+| name             | string               | true                | -                |
+| value            | string               | true                | -                |
+| type             | INPUT_TYPES (one of) | false               | INPUT_TYPES.TEXT |
```

Comment:
sorry to the a problem, but can we have the non required alphabetically ordered? 😄

Reply:
Never apologise for that 😄